packages/manager/apps/vrack-services/src/data/vrack-services/vs-api-utils.ts

```diff
@@ -40,6 +40,13 @@ export const useVrackServicesList = (refetchIntervalTime = 5000) => {
   });
 };
 
+export const REGEX =
```

Comment:
```suggestion
export const CIDR_REGEX =
```

Reply:
Since its in util file lets keep it with generic name.